cartographer/io/internal/pbstream_migrate.cc

```diff
@@ -21,19 +21,18 @@
 #include "gflags/gflags.h"
 #include "glog/logging.h"
 
-DEFINE_bool(migrate_grid_format, false,
-            "Set if the submap data of the input pbstream uses the old "
-            "probability grid format.");
+DEFINE_bool(include_unfinished_submaps, true,
+            "Whether to write to include unfinished submaps in the output.");
```

Comment:
Nit: remove either `to write` or `to include`?

Reply:
Done.